src/components/bottomSheet/types.d.ts

```diff
@@ -140,7 +141,23 @@ export type BottomSheetProps = {
    * @type React.ReactNode[] | React.ReactNode
    */
   children: (() => React.ReactNode) | React.ReactNode[] | React.ReactNode;
-} & BottomSheetAnimationConfigs;
+
+  // accessibility
+  /**
+   * Enables announcement of the snap position change on screen readers.
+   * @type boolean
+   * @default true
+   */
+  announceChangeForAccessibility?: boolean;
+  /**
+   * Callback on snap position change that announces the returned string on screen readers.
+   * @type string | ((positionInScreen: string) => string)
+   */
+  accessibilityPositionChangeAnnouncement?:
```

Comment:
why not just to having a prop for text with placeholder like: 
```ts
// an announcement text with sheet change position
// example: `Bottom sheet snapped to POSITION% of the screen`
accessibilityPositionAnnouncement?: string
```

Reply:
I think that having a `@type string | ((positionInScreen: string) => string)` is a bit easier to use but let me know if you would really prefer the placeholder approach :)

Reply:
it's alright